packages/interfaces/src/relay.ts

```diff
@@ -14,7 +14,7 @@ import type { ISender } from "./sender.js";
 export interface IRelayAPI {
   readonly gossipSub: GossipSub;
   start: () => Promise<void>;
-  getMeshPeers: (topic?: TopicStr) => PeerIdStr[];
```

Comment:
Oh, you changed the API? Why? Previous API seems fine.

Reply:
I would vote to make `Relay.pubsubtopics` `public` instead.
No need for `getAllMeshPeers`.
One should be able to use both `pubSubtopics` and `getMeshPeers` in combination to get the info.
I am not convinced that anyone would use `getAllMeshPeers` because getting a peer in a mesh without knowing what pubsub topic you are interested in seems pointless.